metricbeat/module/dropwizard/collector/_meta/data.json

```diff
@@ -1,51 +1,34 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
+    "@timestamp": "2019-03-01T08:05:34.853Z",
     "dropwizard": {
         "testnamespace": {
-            "my_gauge": {},
-            "my_histogram": {
-                "count": 0,
-                "max": 0,
-                "mean": 0,
-                "min": 0,
-                "p50": 0,
-                "p75": 0,
-                "p95": 0,
-                "p98": 0,
-                "p99": 0,
-                "p999": 0,
-                "stddev": 0
+            "_namespace": "testnamespace",
```

Comment:
This should not be here anymore I would assume?

Reply:
I thought that we could maintain it just for compatibility but I can remove it, of course.

Reply:
I don't think this was in the previous events.